projects/go-lib/src/lib/components/go-copy/go-copy.component.scss

```diff
@@ -0,0 +1,4 @@
+.hideTextArea {
+  position: absolute;
+  left: 99999px;
```

Comment:
I really hate this, but I'm not sure of a better way to do this as doing display:none or setting the height and width to 0 don't work.

Reply:
Yeah, that makes sense if we want it to work in various browsers.